CMakeLists.txt

```diff
@@ -579,12 +580,19 @@ if(UMF_USE_MSAN)
                     "prevent reporting false-positives")
     add_sanitizer_flag(memory)
 endif()
+
 # Fuzzer instrumentation for the whole library
-if(UMF_BUILD_FUZZTESTS
-   AND CMAKE_CXX_COMPILER_ID MATCHES "Clang"
-   AND LINUX)
-    add_compile_options("-fsanitize=fuzzer-no-link")
-    add_link_options("-fsanitize=fuzzer-no-link")
+if(UMF_BUILD_FUZZTESTS)
+    if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND LINUX)
```

Comment:
just asking: why changing `MATCHES` to `STREQUAL`? if we run this on clang++ will this match? should it even be run on clang++?

Reply:
thanks, this should be reverted to use MATCHES

Reply:
done, but please notice that I moved here a code from test/CMakeLists.txt where STREQUAL was used